.gitignore

```diff
@@ -128,8 +131,50 @@ venv.bak/
 .dmypy.json
 dmypy.json
 
+# pytype static type analyzer
+.pytype/
+
+# Cython debug symbols
+cython_debug/
+
 # Pyre type checker
 .pyre/
 
 scripts/minio-conf/
 scripts/minio-data/
+
+## Pycharm ##
+
+.idea/
+.idea_modules/
+*.iml
+modules.xml
+*.ipr
+
+### VisualStudioCode ###
+.vscode/
+!.vscode/settings.json
+!.vscode/tasks.json
+!.vscode/launch.json
+!.vscode/extensions.json
+*.code-workspace
+
+### VisualStudioCode Patch ###
+# Ignore all local history of files
+.history
+.ionide
+
+# Support for Project snippet scope
+!.vscode/*.code-snippets
+
+# temporary files which can be created if a process still has a handle open of a deleted file
+.fuse_hidden*
+
+# KDE directory preferences
+.directory
+
+# Linux trash folder which might appear on any partition or disk
+.Trash-*
+
+# .nfs files are created when an open file is removed but is still being accessed
+.nfs*
```

Comment:
@saltgen - could you please include newline at the end?

Reply:
Added, change pushed